spec/example_app/db/migrate/20170224024549_add_example_time_to_customer.rb

```diff
@@ -0,0 +1,5 @@
+class AddExampleTimeToCustomer < ActiveRecord::Migration[4.2]
+  def change
+    add_column :customers, :example_time, :time
```

Comment:
What do you think about doing `birth_date` and `birth_time` to be consistent?

(It's the closest I can think which might make sense and avoid us having an `example_time` field.)